public/robots.txt

```diff
@@ -0,0 +1 @@
+# See https://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
```

Comment:
Whats this file about?

Reply:
This file is used primarily to manage web crawler (robots) traffic to your site, and give certain instructions about which pages should or should not be viewed. This could be used to exclude certain pages which may be duplicates on our site from being crawled to avoid affecting our SEO scores due to duplicated content.